src/BinSkim.Driver/AnalysisSummaryExtractor.cs

```diff
@@ -40,7 +40,7 @@ public static AnalysisSummary ExtractAnalysisSummary(SarifLog sarifLog, AnalyzeO
                 ToolVersion = tool.Driver.Version,
                 NormalizedPath = string.Join(";", options.TargetFileSpecifiers.Select(p => System.IO.Path.GetDirectoryName(p)).Distinct()),
                 SymbolPath = options.SymbolsPath,
-                FileAnalyzed = artifacts.Count,
+                FileAnalyzed = artifacts?.Count ?? 0,
                 // FileNotAnalyzed =
```

Comment:
>// FileNotAnalyzed = [](http://example.com/codeflow?start=16&length=20)

can you remove this please? it isn't helpful to have it here. 

please add a TODO, or just open an issue

Reply:
created a work item for that.